src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -434,7 +434,9 @@ def load_arguments(self, _):
             c.argument('data_caching', options_list=['--data-disk-caching'], nargs='+',
                        help="storage caching type for data disk(s), including 'None', 'ReadOnly', 'ReadWrite', etc. Use a singular value to apply on all disks, or use '<lun>=<vaule1> <lun>=<value2>' to configure individual disk")
             c.argument('ultra_ssd_enabled', arg_type=get_three_state_flag(), min_api='2018-06-01',
-                       help='(PREVIEW)Enables or disables the capability to have 1 or more managed data disks with UltraSSD_LRS storage account')
+                       help='(PREVIEW) Enables or disables the capability to have 1 or more managed data disks with UltraSSD_LRS storage account')
+            c.argument('use_local_os_disk', arg_type=get_three_state_flag(), min_api='2018-06-01',
```

Comment:
I would suggest just name it as `--ephemeral-os-disk`  which is the market term. The help text can be `allows you to create an OS disk directly on the host node, providing local disk performance and faster VM/VMSS reimage time.` and remove the link